scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/EHSMAttestationService.scala

```diff
@@ -41,13 +51,17 @@ class EHSMAttestationService(kmsServerIP: String, kmsServerPort: String,
   override def setPolicy(policy: JSONObject): String = "true"
 
   def getQuoteFromServer(): String = {
+    // TODO Get qutoe from ehsm
     "test"
   }
 
   override def attestWithServer(quote: String): (Boolean, String) = {
     // TODO nonce
     val nonce: String = "test"
-    require(quote != null, "quote should be specified")
```

Comment:
`Log4Error.invalidInputError(` you can use this one.

Reply:
Addressed